protocols/decentralgames.eth/overview.md

```diff
@@ -0,0 +1,42 @@
+# Decentral Games
+
+Decentral Games builds games that give players economic freedom through aligned incentives, self-custody and delegation of yield-bearing metaverse assets. 
+
+# Overview
+
+Decentral Games ICE Poker is a free metaverse poker game where players earn tokens by completing daily challenges and competing against each other on a daily leaderboard. 
+
+* ICE is an in-game currency that incentivizes player liquidity and enables upgrades of in-game items.
+* DG is a utility token spent to activate and upgrade ICE Wearables. xDG (staked DG) is the governance token that grants access to premium ICE Poker Guild tools, as well as votes to modify economic incentives and govern the Decentral Games treasury.
+
+## Play and Earn
+
+In order to participate, players need to obtain at least one ICE Wearable either through purchase or delegation. Players are allocated a set amount of Chips which are used to play ICE Poker. Read more about ICE NFT Wearables and how players can earn [here](https://docs.decentral.games/)
+
+# Governance Process
+
+Decentral Games governance proposals are submitted on Snapshot. This is where voting takes place as well. The xDG token acts as the governance instrument, enabling the community to vote on new game development, effectively allocate the treasury funds, and adjust ecosystem parameters in order to manage risk, as well as respond to changing market demands and conditions. 
+
+This makes the governance of the platform directly responsive to our most active users and brings Decentral Games closer to being a public good operated by the community members who value it most. DG holders who stake in the governance contract will be able to decide on: 
+
+* ICE burn prices for all wearable ranks
+* DG prices for all wearable ranks
+* ICE bonuses for wearable ranks
+* DG activation price for ICE wearables
+* Challenges and difficultly variables
+* Leaderboard multipliers for each tier
+* Base player/owner splits
+
+In order to submit a proposal on Snapshot, one must have 100k xDG. Proposals pass/fail via simple majority, but 10M xDG voting in favor is required to reach a quorum.
+
```

Comment:
Can you please add ### Join Decentral Games section with the following links , 
Something like _please reach out to us at telegram or discord_
![image](https://user-images.githubusercontent.com/41290852/155001883-3b86a83a-1bf5-412c-a5e1-c5196ccb35b8.png)